apps/portals/vite.config.ts

```diff
@@ -0,0 +1,48 @@
+import react from '@vitejs/plugin-react'
+import { resolve } from 'path'
+import { defineConfig } from 'vite'
+import svgr from 'vite-plugin-svgr'
+
+export default defineConfig({
+  envDir: './src/config',
```

Comment:
Tell Vite to pull the `.env` file from the current portal config. The rest of the config is the same as the others.